src/main/java/nextstep/subway/dto/LineRequest.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.subway.dto;
+
+import nextstep.subway.domain.Line;
+
+public class LineRequest {
+    private String name;
+
+    private String color;
+
+    public String getName() {
+        return name;
+    }
+
+    public String getColor() { return color; }
```

Comment:
```suggestion
    public String getColor() { 
        return color; 
    }
```

한 라인으로 작성한 것은 의도된 내용일까요? 🤔

Reply:
굳이 라인수가 길어질 필요가 없을것 같아
Settings > Editor > General > Code Folding > One-line methods를
보통 설정해놓고 작업하다보니 한 줄로 저장될때가 많습니다.

이 경우, 한 라인으로 작성하지 않는게 주로 많이 쓰이는 컨벤션인가요??